ast/Trees.cc

```diff
@@ -1160,8 +1165,10 @@ string OptionalArg::showRaw(const core::GlobalState &gs, int tabs) {
     buf << nodeName() << "{" << '\n';
     printTabs(buf, tabs + 1);
     buf << "expr = " + expr->showRaw(gs, tabs + 1) << '\n';
-    printTabs(buf, tabs + 1);
-    buf << "default_ = " + default_->showRaw(gs, tabs + 1) << '\n';
+    if (default_) {
+        printTabs(buf, tabs + 1);
+        buf << "default_ = " + default_->showRaw(gs, tabs + 1) << '\n';
+    }
```

Comment:
Do you know why some formatting functions use `stringstream` and some use `fmt::memory_buffer`?

Reply:
`stringstream` was what existed before. I'm assuming someone who was more clever than I used `fmt::memory_buffer` and didn't do the effort to make it consistent so now we have two ways of doing it.